### ISSUE TITLE
Issue created in ga4gh/schemas to catch results

### ISSUE THREAD

**Issue by kozbo** [Ensembl/schemas#8]
https://github.com/ga4gh/schemas/issues/822

Merging in these use cases will require a little work. Previous development was not strictly use-case driven. Some (closed) issues were driven buy use cases and we should mine those cases out of the old issues and add them into the master use-case list.

(I wasn't sure where to put this link other than in another issue. I think that this issue can be closed once the fork is merged to /ga4gh/schemas)
